owasm/chaintests/gold_price/src/lib.rs

```diff
@@ -0,0 +1,31 @@
+use borsh::{BorshDeserialize, BorshSerialize};
+use owasm::{execute_entry_point, ext, oei, prepare_entry_point};
+
+#[derive(BorshDeserialize)]
+struct Input {
+    symbol: String,
+    multiplier: u64,
+}
+
+#[derive(BorshSerialize)]
+struct Output {
+    px: u64,
+}
+
+#[no_mangle]
+fn prepare_impl(input: Input) {
+    // Gold price data source
+    oei::request_external_data(5, 1, input.symbol.as_bytes());
+    // Binance data source
+    oei::request_external_data(6, 2, input.symbol.as_bytes());
```

Comment:
I think we can use existing Binance script and send "ATOM".as_bytes() here instead